rustls-libssl/tests/runner.rs

```diff
@@ -88,6 +88,29 @@ fn client() {
     assert_eq!(openssl_secure_output, rustls_secure_output);
 }
 
+#[test]
+#[ignore]
+fn client_real_world() {
+    let openssl_output = Command::new("target/client")
+        .env("LD_LIBRARY_PATH", "")
+        .env("NO_ECHO", "1")
+        .args(&["example.com", "443", "default"])
```

Comment:
After a rebase, this can become:
```suggestion
    let openssl_output = Command::new("tests/maybe-valgrind.sh")
        .env("LD_LIBRARY_PATH", "")
        .env("NO_ECHO", "1")
        .args(&["target/client", "example.com", "443", "default"])
```
to let valgrind work. (Ditto below)

Reply:
good catch, thank you.